org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/codemining/CodeMiningFragment.xtend

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright (c) 2018 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
```

Comment:
not exported in manifest?

Reply:
make it x-internal first?

Reply:
yes

Reply:
yes. i wonder why this works at all in domainmodel